docs/docs/getting-started/componentDefinition.mdx

```diff
@@ -0,0 +1,13 @@
+A component is a Helm Chart that can be installed on an Airy Cluster in order to extend its functionality.
+
+Components can be characterized by the functionality they enable:
+
+1. UI Components that provide new user interfaces in Airy
+2. Connector Components that connect one or more third-party services with Airy to exchange data. 
```

Comment:
As `sources` is everywhere in the docs, I recommend that we add a note here that the `sources` fit also under connectors.

Reply:
We will omit these details about the **kinds** of connectors for now. As we discussed, this is something for a more extensive discussion between product and engineering to unify our language.